pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.smallrye</groupId>
+    <artifactId>smallrye-parent</artifactId>
+    <version>6</version>
+  </parent>
+
+  <groupId>io.smallrye.event-sourcing</groupId>
+  <artifactId>smallrye-event-sourcing</artifactId>
+  <version>0.0.10-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>Smallrye Event Sourcing</name>
+  <description>An event sourcing proof of concept built on MicroProfile Reactive Messaging</description>
+
+  <inceptionYear>2019</inceptionYear>
+  <organization>
+    <name>SmallRye</name>
+    <url>https://wwww.smallrye.io</url>
+  </organization>
+
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+      <comments>A business-friendly OSS license</comments>
+    </license>
+  </licenses>
+
+  <developers>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:smallrye/smallrye-event-sourcing.git</connection>
+    <developerConnection>scm:git:git@github.com:smallrye/smallrye-event-sourcing.git</developerConnection>
+    <url>https://github.com/smallrye/smallrye-event-sourcing</url>
+    <tag>HEAD</tag>
+  </scm>
+
+
+  <properties>
+  </properties>
+
+  <modules>
+  </modules>
+
+  <dependencyManagement>
+    <dependencies>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+  </dependencies>
+
+  <build>
+    <defaultGoal>clean install</defaultGoal>
+
+    <pluginManagement>
```

Comment:
I think the `<pluginManagement>` section can be removed as it's present in the parent

Reply:
Done.